lib/uci.c

```diff
@@ -158,11 +158,13 @@ uc_uci_cursor(uc_vm_t *vm, size_t nargs)
 {
 	uc_value_t *cdir = uc_fn_arg(0);
 	uc_value_t *sdir = uc_fn_arg(1);
+	uc_value_t *c2dir = uc_fn_arg(2);
```

Comment:
Please document the new parameter in the comment block above, using something like this:

```
 * @param {string} [override_dir]
 * If set, overlay copies of modified configuration files are read from and
 * written to this directory instrad of the primary configuratio directory.
 * 
 * This can be used to override specific config files at runtime without
 * touching them on primary storage.
```

Reply:
Just to understand the semantics; libuci now always check `/var/run/uci` by default and if a given config exists there it is is read from and written to there?

Reply:
Got it, thanks. Will merge this once green.